README.md

```diff
@@ -26,30 +26,34 @@ The pipeline exposes the following standard settings:
 * `pipe_events.pipeline_bucket`: GCS bucket where all final files generated by this pipeline may be stored to.
 * `pipe_events.pipeline_dataset`: BigQuery dataset containing various tables used in this pipeline.
 
-In addition to this, the following custom settings are required for this
-pipeline, and come with default values:
+In addition to this, the following custom settings are required for this pipeline, and come with default values:
 
 * `INFERENCE_BUFFER_DAYS`: Global airflow variable which determines the amount of days to reprocess because nnet scores might be avaialble. Defaults to `7`.
 * `pipe_events.events_dataset`: BigQuery dataset which will contain the published events. Defaults to `EVENTS_DATASET`.
 * `pipe_events.source_dataset`: BigQuery dataset which contains the different tables that are read to produce the summarized events. Defaults to `PIPELINE_DATASET`.
 * `pipe_events.gaps.source_table`: BigQuery table to read gaps from. Defaults to `position_messages_`.
 * `pipe_events.gaps.events_table`: BigQuery table to publish gap events to. Defaults to `published_events_gaps`.
 * `pipe_events.gaps.segment_vessel`: BigQuery table containing segment information, used to calculate gaps. Defaults to `segment_vessel`.
+* `pipe_events.gaps.vessel_info`: BigQuery table to read vessel information from. Defaults to `vessel_info`.
 * `pipe_events.gaps.gap_min_pos_count`: Only consider segments with a given minimum amount of positions. Defaults to `3`.
 * `pipe_events.gaps.gap_min_dist`: Minimum distance to shore to consider the transponder off events. Defaults to `10000`.
 * `pipe_events.encounters.source_table`: BigQuery table containing the raw encounters to read from. Defaults to `encounters`.
+* `pipe_events.encounters.vessel_info`: BigQuery table to read vessel information from. Defaults to `vessel_info`.
 * `pipe_events.encounters.events_table`: BigQuery table to publish the encounters to. Defaults to `published_events_encounters`.
 * `pipe_events.anchorages.source_table`: BigQuery table containing the raw port events to read from. Defaults to `port_events_`.
+* `pipe_events.anchorages.vessel_info`: BigQuery table to read vessel information from. Defaults to `vessel_info`.
 * `pipe_events.anchorages.events_table`: BigQuery table to publish the anchorages to. Defaults to `published_events_ports`.
+* `pipe_events.anchorages.anchorages_dataset`: BigQuery dataset which contains the named anchorages table. Defaults to `gfw_research`.
```

Comment:
@andres-arana any reason why you are splitting the anchorages table into two variables? dataset and table? I would only use one variable that has both `dataset.table` unless you are using them separately. Having two variables is error prone since you may change the table and forget to change the dataset.

Reply:
@enriquetuya just for consistency. Every other table is referenced that way, and having only one which is joined seems a bit of, although I agree with you and *every* other variable should be a full-qualified table name.